bluemira/geometry/parameterisations.py

```diff
@@ -986,6 +986,191 @@ def _make_control_points(p0, p3, theta0, theta3, l_start, l_end):
         return p1, p2
 
 
+class PictureFrameTools:
+    """
+    Curved Pictureframe parameterisation with straight (flat) inner limb
+
+    Parameters
+    ----------
+    var_dict: Optional[dict]
+        Dictionary with which to update the default values of the parameterisation.
+
+    """
```

Comment:
`PictureFrameTools` is not a inheriting from `GeometryParametrization` and does not have an `__init__` finction that would contain `var_dict`, are you sure this description is meant for `PictureFrameTools`?

Reply:
updated